CRM/Contribute/Page/ContributionRecurPayments.php

```diff
@@ -53,6 +54,9 @@ private function loadRelatedContributions() {
       $this->insertStatusLabels($contribution);
       $this->insertContributionActions($contribution);
 
+      if ($contribution['is_test']) {
+        $contribution['financial_type'] = '(' . ts('Test' . ') ') . $contribution['financial_type'];
```

Comment:
Can you change the ts(test) bit to: `ts(' (test)')` which I think is more consistent with code elsewhere and with #13698